scipy/stats/_censored_data.py

```diff
@@ -115,6 +115,27 @@ class CensoredData:
     to use `uncensored`, `left` and `right` for uncensored, left-censored and
     right-censored observations, respectively.
 
+    The ``fit`` method of continuous univariate SciPy distributions find
+    the maximum likelihood estimates by minimizing the following function:
+
+    .. math::
+
+        l(x; \pmb{\\theta}) & = \sum_{k=1}^{n_{nc}} 
+                                \log(f(\\text{uncensored[k]}; \pmb{\\theta}))
```

Comment:
To avoid having to define both the notation $\text{uncensored[k]}$ and the numbers of each type of observation, how about writing the sums like $\sum \log(f(k_u))$ and instead defining $k_u$ as the uncensored observations. (Technically, $u$ is the index, and it takes on the values corresponding with the uncensored observations.)